src/web/src/components/utilBar/UtilBar.js

```diff
@@ -48,6 +49,7 @@ const UtilBar = () => {
   };
   return (
     <div className={classes.container}>
+      <UserButton/> 
```

Comment:
Put this component `<UserButton />` in `renderBasedOnPathname()` function and have it returned if pathname is `/expense/w*`. Because, show member functionality is only available when in an `expense` (similar to group in discord)